docs/_pages/getting_started.md

```diff
@@ -75,46 +86,47 @@ If you see the same output as above, you're ready to build your Slack app!
 
 ## Sending a message with the Web API
 
-In this guide we'll post a simple message that contains the current time. We'll also follow
-the best practice of keeping secrets outside of your code (do not hardcode sensitive data).
+In this guide we'll post a simple message that contains the current time. We'll also follow the best practice of keeping
+secrets outside of your code (do not hardcode sensitive data).
 
-Store the access token in a new environment variable. The following example works on Linux and MacOS;
-but [similar commands are available on Windows](https://superuser.com/a/212153/94970). Replace the
-value with OAuth Access Token that you copied above.
+Store the access token in a new environment variable. The following example works on Linux and MacOS; but [similar
+commands are available on Windows](https://superuser.com/a/212153/94970). Replace the value with OAuth Access Token that
+you copied earlier.
 
 ```shell
-$ export SLACK_ACCESS_TOKEN=xoxp-...
+$ export SLACK_TOKEN=xoxp-...
```

Comment:
Do we really want to remove the word `access`? Not opposed to this, but since it's an "OAuth _Access_ Token", there's a certain level of continuity by using `SLACK_ACCESS_TOKEN`

Reply:
its `SLACK_TOKEN` all over the `web_api.md` page, so i made this change for consistency. i don't feel like the word `access` conveys any real meaning, because for the purposes of this document, there's only one token. i think it starts to matter when you talk about how OAuth works. open to changing that though.